src/main/java/org/openstreetmap/atlas/checks/distributed/IntegrityCheckSparkJob.java

```diff
@@ -211,7 +133,7 @@ public String getName()
         return "Integrity Check Spark Job";
     }
 
-    @SuppressWarnings({ "rawtypes" })
+    @SuppressWarnings({ "rawtypes", "unchecked" })
```

Comment:
What are these warnings?

Reply:
I'm not sure what rawtypes is for, but was already there. unchecked is for unchecked casts, that are happening when getting values from the command map.